person.rb

```diff
@@ -24,6 +25,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ]  I'll suggest you to implement the method to add rentals as below so that the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. take a look at the code below:
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
@RolandM99 thank you, I have implemented all of the required changes.